### ISSUE TITLE
[Enhancement][Alert] Add default vocalisation on "info" status

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#806]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iOS

### OS Version

26

### App version

2.0.0

### Describe the issue

For alert components, "warning" and "error" status are correctly vocalised with screen reader. But "info" status should also be vocalized as it can be useful for the user.

I have no idea about others status "neutral", "accent" and "positive", by default no label but maybe optional, depending on context?

### Expected Behavior

"info" status should be vocalized "information"

**Comment by AhmedAmineZr** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (733)
Version: 3.0.0
Build type: Alpha
Build details (GitHub): 806
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by AhmedAmineZr** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (733)
Version: 3.0.0
Build type: Alpha
Build details (GitHub): 806
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
@AhmedAmineZr "Info" status is vocalized on alert message but not on inline alert

**Comment by AhmedAmineZr** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (734)
Version: 3.0.0
Build type: Alpha
Build details (GitHub): 806
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by AhmedAmineZr** [comment]
New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (734)
Version: 3.0.0
Build type: Alpha
Build details (GitHub): 806
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
fixed
